pkg/client/kubernetes/scaling.go

```diff
@@ -49,7 +50,19 @@ func ScaleEtcd(ctx context.Context, c client.Client, key client.ObjectKey, repli
 		},
 	}
 
-	return scaleResource(ctx, c, etcd, int32(replicas))
+	if err := c.Get(ctx, key, etcd); err != nil {
+		return err
+	}
+
+	patch := client.MergeFrom(etcd.DeepCopy())
+	if etcd.Annotations == nil {
+		etcd.SetAnnotations(make(map[string]string))
+	}
+
+	etcd.Annotations[gardencorev1beta1constants.GardenerOperation] = gardencorev1beta1constants.GardenerOperationReconcile
```

Comment:
Should we also set the timestamp annotation here (since we usually do this when setting the operation annotation)?

Reply:
Thanks, done in c4b936f46ab4ce221cd970af0ddea55ead51cea5.